www/grid_view/src/module/changeinfo.directive.coffee

```diff
@@ -0,0 +1,8 @@
+class Changeinfo extends Directive
```

Comment:
You should call it GridViewChangInfo. If possible, I would like to reuse the changedetails directive